hawkbit-dmf/hawkbit-dmf-api/src/main/java/org/eclipse/hawkbit/dmf/json/model/DmfActionUpdateStatus.java

```diff
@@ -34,10 +34,10 @@ public class DmfActionUpdateStatus {
     private Long softwareModuleId;
 
     @JsonProperty
-    private Integer code;
+    private List<String> message;
 
     @JsonProperty
-    private List<String> message;
+    private int code;
```

Comment:
Please revert to type 'Integer' since it's an nullable field. Otherwise it's never an empty Optional in:

@JsonIgnore
    public Optional<Integer> getCode() {
        return Optional.ofNullable(code);
    }